src/main/java/seedu/realodex/logic/parser/DeleteCommandParser.java

```diff
@@ -12,21 +12,45 @@
  */
 public class DeleteCommandParser implements Parser<DeleteCommand> {
 
+    public static final String INDEX_AND_NAME_PROVIDED = "Please provide either an index or a name, not both.";
+    public static final String NO_FIELDS_PROVIDED = "Please provide either an index or a name.";
+
     /**
      * Parses the given {@code String} of arguments in the context of the DeleteCommand
      * and returns a DeleteCommand object for execution.
      * @throws ParseException if the user input does not conform the expected format
      */
     public DeleteCommand parse(String args) throws ParseException {
         ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_NAME);
+        String trimmed = args;
+
+        // Parse name if present
```

Comment:
I think should add `argMultimap.verifyNoDuplicatePrefixesFor` check for double `n/`

Reply:
Resolved thanks!